src/index.js

```diff
@@ -72,21 +75,28 @@ function installRouterPrefetch(
           observer.unobserve(this.$el)
         }
       },
-      getComponents() {
-        return this.$router.getMatchedComponents(this.to).filter(Component => {
-          return Component.cid === undefined && typeof Component === 'function'
-        })
+      getRouteComponents(route) {
```

Comment:
`vue-router` doesn't expose a `getMatchedComponents()` method anymore, so we need to define it by ourselves.